packages/web3-eth/test/integration/rpc.block.test.ts

```diff
@@ -29,17 +29,17 @@ import {
 	createNewAccount,
 	isIpc,
 	isWs,
+	createTempAccount,
 } from '../fixtures/system_test_utils';
 import { BasicAbi, BasicBytecode } from '../shared_fixtures/build/Basic';
 import { toAllVariants } from '../shared_fixtures/utils';
 import { sendFewTxes, validateBlock, validateTransaction } from './helper';
 
 describe('rpc with block', () => {
 	let web3Eth: Web3Eth;
-	let accounts: string[] = [];
 	let clientUrl: string;
 
-	let contract: Contract<typeof BasicAbi>;
+	let contractInstance: Contract<typeof BasicAbi>;
```

Comment:
`contractInstance = new Contract(...)`
so we received an instance of the contract. that's why I called this variable `contractInstance`

Reply:
If this is the case, let us keep with the convention of using camleCase for class instances.
Actually, the camleCase indicate an instance. Or else, we need to rename all varaiables like `web3Eth`, `tempAcc`... in this file and in all other files, because they are also all "instances".
Thanks,

Reply:
renamed. check plz